lang/cs/Org.Apache.REEF.Network/Group/Topology/FlatTopology.cs

```diff
@@ -103,7 +106,12 @@ public IConfiguration GetTaskConfiguration(string taskId)
 
             if (OperatorSpec is BroadcastOperatorSpec<T>)
             {
-                BroadcastOperatorSpec<T> broadcastSpec = OperatorSpec as BroadcastOperatorSpec<T>;
+                var broadcastSpec = OperatorSpec as BroadcastOperatorSpec<T>;
+
+                confBuilder.AddConfiguration(broadcastSpec.PipelineDataConverter.GetConfiguration());
+                confBuilder.BindImplementation(typeof(IPipelineDataConverter<T>), broadcastSpec.PipelineDataConverter.GetType())
```

Comment:
Use this line instead. 
 .BindImplementation(GenericType<IPipelineDataConverter<T>>.Class, GenericType<DefaultPipelineDataConverter<T>>.Class)